app/assets/javascripts/app/reservation_time_checker.coffee

```diff
@@ -1,8 +1,12 @@
 class window.ReservationTimeChecker
   constructor: (@selector, @alertId = 'duration-alert')->
-    @initAlert()
-    @respondToChange()
+    if @validPage()
+      @initAlert()
+      @respondToChange()
 
+  validPage: ->
+    # These variable are not set on the admin reservation pages
+    reserveInterval? && reserveMinimum? && reserveMaximum?
```

Comment:
Would it make sense to change the method `validPage` to `adminReservationPage?`? 

So you'd have something like...

```coffeescript
class window.ReservationTimeChecker
  constructor: (@selector, @alertId = 'duration-alert')->
    unless @adminReservationPage()
      @initAlert()
      @respondToChange()

  adminReservationPage: ->
    reserveInterval? && reserveMinimum? && reserveMaximum?
```